src/test/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImplTestIT.java

```diff
@@ -53,6 +53,74 @@ public void resetMockServerClient() {
         mockServerClient.reset();
     }
 
+    @Test
+    void getPgConsentByType() throws JsonProcessingException {
+        Consent consent = consentsMock.getPgTosConsentResponseMock();
+        String response = objectMapper.writeValueAsString(consent);
+        mockServerClient.when(request().withMethod("GET").withPath(path + "/TOS"))
```

Comment:
Check the path if it is the same of the micro-service api

Reply:
done